alarm/models.py

```diff
@@ -66,4 +84,17 @@ def refresh_candle_data(cls, trade_pair, high_price, low_price):
         return candle
 
     def __str__(self):
-        return f"{self.trade_pair}: {self.low_price}-{self.high_price}"
+        return f"{self.low_price}-{self.high_price}"
+
+
+class ThresholdBrake(models.Model):
+    threshold = models.ForeignKey(Threshold, on_delete=models.CASCADE)
+    happened_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.threshold}"
+
+    def delete_old_threshold_brake(self):
```

Comment:
There is a lot of threshold brakes. Please pay attention to this. It is very easy to fix and quite irritating to review.

Reply:
++